rpm/README.md

```diff
@@ -0,0 +1,87 @@
+# HOW-TO Maintain RPM lists for containers when adding new target platforms and architectures
+
+## Overview
+
+The file `rpms/BUILD.bazel` provides a list pinned rpms to be built in each category of CDI containers
+
+These are maintained with bazeldnf with rpm repos and names of packages  specified in the following three files:
+
+ 1. `.bazelrc`
+ 2. `repo.yaml`
+ 3. `hack/build/rpm-deps.sh` 
+
+Then, running `make rpm-deps` _should_ invoke bazeldnf through the bazel cdi builder container to populate the `rpm/BUILD.bazel` file 
+
+Once rpm/BUILD.bazel is populated, it is used and re-used, pinning the rpms to be built into run & test container images to specific versions. 
+This is why it is checked in to github per CDI release process. 
+
+## Prerequisites and Caveats
```

Comment:
Not sure this is a great header for what follows.  Seems like you are describing how to use bazeldnf without the builder container?

Reply:
No, in all cases the `make` target `rpm-deps` invokes the builder container. It's just that you need `bazeldnf` installed on the host for it to work, it seems.  `bazeldnf` is not pre-installed in the builder container.

Reply:
Ah!  so after `make rpm-deps` is complete as a result of your https://github.com/kubevirt/containerized-data-importer/blob/main/WORKSPACE#L83-L95, we find: 

```
[cfillekes@m1325001 containerized-data-importer]$ podman ps
CONTAINER ID  IMAGE                                                    COMMAND               CREATED        STATUS        PORTS       NAMES
3c4b6fed495d  icr.io/kubevirt/kubevirt-cdi-bazel-builder:native-s390x  hack/build/bazel-...  4 minutes ago  Up 4 minutes              kubevirt-cdi-volume-bazel-server
[cfillekes@m1325001 containerized-data-importer]$ podman exec -it 3c4b6fed495d /bin/bash
go version go1.21.5 linux/s390x
[root@m1325001 containerized-data-importer]# which bazeldnf
/usr/bin/which: no bazeldnf in (/root/.local/bin:/root/bin:/gimme/.gimme/versions/go1.21.5.linux.s390x/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin)
[root@m1325001 containerized-data-importer]# find / -name bazeldnf
find: ‘/sys/fs/pstore’: Permission denied
find: ‘/sys/fs/bpf’: Permission denied
/root/.cache/bazel/_bazel_root/e03bf9038fc5089c0dbc8615812d9838/execroot/__main__/bazel-out/host/bin/external/bazeldnf
/root/.cache/bazel/_bazel_root/e03bf9038fc5089c0dbc8615812d9838/execroot/__main__/bazel-out/host/bin/external/bazeldnf/pkg/api/bazeldnf
/root/.cache/bazel/_bazel_root/e03bf9038fc5089c0dbc8615812d9838/execroot/__main__/bazel-out/s390x-fastbuild/bin/bazeldnf.bash.runfiles/__main__/external/bazeldnf
/root/.cache/bazel/_bazel_root/e03bf9038fc5089c0dbc8615812d9838/execroot/__main__/bazel-out/s390x-fastbuild/bin/bazeldnf.bash.runfiles/bazeldnf
/root/.cache/bazel/_bazel_root/e03bf9038fc5089c0dbc8615812d9838/execroot/__main__/external/bazeldnf
/root/.cache/bazel/_bazel_root/e03bf9038fc5089c0dbc8615812d9838/external/bazeldnf
/root/.cache/bazel/_bazel_root/e03bf9038fc5089c0dbc8615812d9838/external/bazeldnf/pkg/api/bazeldnf
```


let me fix that.